### PR TITLE
Pull 2019-02-06T16-42 Recent NVIDIA Changes

### PR DESCRIPTION
**Pull request by gklimowicz**

### CODE REVIEW

File: runtime/libpgmath/lib/common/debug_prn.h [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    snprintf(buffer, 15+1+1, "%15s:", filename + THIS_MAX((int)strlen(filename) - 15, 0));
+    #undef THIS_MAX
+    // print line number
+    snprintf(tmp, 5+1+1+1, "%5d: ", linenum);
+    strncat(buffer, tmp, 5+1+1);
+    // print status flags
+    print_status_flags(buffer);
+    strncat(buffer, ": ", 2);
+    // print variable name
+    snprintf(tmp, 14+1, "%14s:", varname);
```
Comment (sscalpone) [inline runtime/libpgmath/lib/common/debug_prn.h]: bug? 14 + ':' + null =16

File: runtime/libpgmath/lib/common/debug_prn.h [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    {
+        case sizeof(float):
+        {
+            float    fval = *((float*)pvar);
+            double   dval = (double)fval;
+            unsigned uval = *((unsigned*)pvar);
+            snprintf(buffer1, 150, "%s:         %08X == %-20g == %-25a == %d", buffer, uval, dval, fval, uval);
+        }
+            break;
+default:
```
Comment (sscalpone) [inline runtime/libpgmath/lib/common/debug_prn.h]: Is this flang formatted?

File: runtime/libpgmath/lib/common/debug_prn.h (hunk 1 above)
Comment (gklimowicz) [reply runtime/libpgmath/lib/common/debug_prn.h]: yes; I will resolve in upcoming PR.

File: runtime/libpgmath/lib/common/debug_prn.h (hunk 2 above)
Comment (gklimowicz) [reply runtime/libpgmath/lib/common/debug_prn.h]: I'll make sure that it is when I fix the above issue.

**Pull request merged**